formats/licence/publisher/details.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "licence_identifier"
```

Comment:
As `licence_identifier` is the only required field, what does the content look like when only this is provided?